### PR TITLE
[riffcpp] Update to 2.2.4

### PR DESCRIPTION
**Pull request by frabert**

### CODE REVIEW

**Comment by vicroms** [comment]
Please update the version inside the `CONTROL` file.

File: ports/riffcpp/portfile.cmake [hunk 1]
```diff
@@ -3,8 +3,8 @@ include(vcpkg_common_functions)
 vcpkg_from_github(
     OUT_SOURCE_PATH SOURCE_PATH
     REPO libdmusic/riffcpp
-    REF  v2.2.2
-    SHA512 2124208e0e75bfe6d96fd177594ea038291e2bbf88c3a1fddb9cede0966168748de43844a379253947797baf184d85ca0fa521edbf9001cf9c5e471356f1603d
+    REF  v2.2.4
```
Comment (vicroms) [inline ports/riffcpp/portfile.cmake]: Need to update version in `CONTROL` file.

**Review by vicroms — CHANGES_REQUESTED** [review]

File: ports/riffcpp/portfile.cmake (hunk 1 above)
Comment (frabert) [reply ports/riffcpp/portfile.cmake]: Oops, I had forgot about that. Should be fixed now!

**Review by vicroms — APPROVED** [review]

**Comment by vicroms** [comment]
Thanks for the PR!

**Pull request merged**